src/hyperdisplay.h

```diff
@@ -4,10 +4,12 @@ SparkFun Master Display Library
 
```

Comment:
White space changes (presumably cause by different tab width settings) make this diff hard to process.